src/main/java/com/appcharge/server/service/AuthService.java

```diff
@@ -2,7 +2,10 @@
 
 import com.appcharge.server.models.auth.AuthResponse;
 import com.appcharge.server.models.auth.AuthenticationRequest;
+import com.appcharge.server.models.auth.OtpAuthResponse;
+import com.appcharge.server.models.auth.OtpAuthenticationRequest;
 
 public interface AuthService {
     AuthResponse authenticatePlayer(AuthenticationRequest authRequest, SecretsService secretsService);
-}
\ No newline at end of file
+    OtpAuthResponse authenticatePlayerOtp(OtpAuthenticationRequest otpAuthRequest);
```

Comment:
it's not otpAuthResponse, it's otpDeepLinkGenerationResponse
otp is not auth. please, change everywhere